user-guide/Advanced_Functionality/Swarming/SwarmingBookings.md

```diff
@@ -0,0 +1,102 @@
+---
+uid: SwarmingBookings
+---
+
+# Swarming bookings
+
+Since 10.4.4, it is possible to swarm bookings. To do so, you need the *Swarming* soft-launch option. 
+From 10.5.1 onwards, the feature is placed behind a different soft-launch option, namely *BookingSwarming*.
+
+Swarming bookings can be done by sending a *SwarmingRequestMessage*, to which you add the new hosting agent ID and booking ID. 
+This can be done as follows:
+
+```csharp
+using System;
+using System.Linq;
+using Skyline.DataMiner.Automation;
+using Skyline.DataMiner.Net.Swarming;
+
+public class Script
+{
+    public void Run(Engine engine)
+    {
+        var newHostingAgent = 123;
+        var bookingId = Guid.Parse("...");
+
+        var request = SwarmingRequestMessage.ForBooking(newHostingAgent, bookingId);
+        var response = engine.SendSLNetMessage(request)?.First() as SwarmingResponseMessage;
+
+        if (response == null)
+        {
+            engine.ExitFail($"Response was null, which was not expected.");
+        }
+
+        foreach (var result in response.SwarmingResults)
+        {
+            if (!result.Success)
+            {
+                engine.ExitFail($"Swarming failed: {result?.Message}.");
+            }
+        }
+    }
+}
+```
+
+> [!NOTE]
+>
+> - In order to swarm a booking, the new hosting agent must be up and running. In case the current hosting agent is unreachable, swarming will still take place but an error will be in the log file. 
+> -  If you swarm a booking which has services or resources linked to it, these will not be swarmed to the new agent. 
```

Comment:
Maybe clarify 'these' means the services and function resources? The booking itself will still be swarmed.